.ci/pitest.sh

```diff
@@ -227,7 +227,7 @@ pitest-coding-require-this-check)
 --list)
   echo "Supported profiles:"
   pomXmlPath="$(dirname "${0}")/../pom.xml"
-  sed -n -e 's/<id>\(pitest-\w\+\)<\/id>/\1/p' < "${pomXmlPath}" | sort
+  sed -n -e 's/^.*<id>\(pitest-[^<]\+\)<\/id>.*$/\1/p' < "${pomXmlPath}" | sort
```

Comment:
Added `^.*` and `.*$` to match less, to get rid of extra whitespace and not to match extra stuff at the end of the line, eg-

Without `^.*` and `.*$`

Input file - 
```
            some more text<id>pitest-abc</id>some extra text
```
Output-
```
            some more textpitest-abcsome extra text
```